docs/user/reference/metrics/event.md

```diff
@@ -215,6 +223,21 @@ auto optEvents = mozilla::glean::views::login_opened.TestGetValue();
 auto events = optEvents.extract();
 ASSERT_EQ(2UL, events.Length());
 ASSERT_STREQ("login_opened", events[0].mName.get());
+
+// Note that the list of extra key/value pairs can be in any order.
```

Comment:
@chutten I'm not sure how best to show what folks need to know so I heavily borrowed from[ a FOG test](https://searchfox.org/mozilla-central/source/toolkit/components/glean/tests/gtest/TestFog.cpp#156) since it was illuminating to me. Is this too much? Not Enough?

Reply:
Well, looping after asserting it's an array of length 1 is a little odd. I'd probably `auto extra = events[0].mExtra[0];` and then straight up assert the key STREQ "source_of_login" and the value STREQ "toolbar"

Reply:
Fair enough. If you think that is adequately descriptive for someone to understand then I am fine with it personally. I was mostly motivated by the lack of a guarantee re ordering, maybe for a C++ dev this is an exercise best left to the reader? I will update this